script.js

```diff
@@ -5,7 +5,336 @@ hamburgerIcon.addEventListener('click', () => {
+    ],
+    description: [
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.",
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    ],
+    featuredImage: [
+      {
+        featuredImageMobile: './assets/recent_works/hm_img.png',
+        featuredImageDesktop: './assets/recent_works/bg-img-desktop.png',
+      },
+      './assets/recent_works/bg-img.png',
+      './assets/recent_works/art.svg',
+      './assets/recent_works/data.svg',
+      './assets/recent_works/website.svg',
+      './assets/popups/snapshootPortfolio.png',
+      './assets/popups/snapshootPortfolio.svg',
+    ],
+    technologies: ['CSS', 'HTML', 'Bootstrap', 'Ruby', 'Ruby on Rails'],
+    button: [
+      'See project',
+      'See live',
+      'See source',
+    ],
+    live: 'https://allenkays.github.io/',
+    source: 'https://github.com/allenkays/Portfolio-setup-and-mobile-first',
+  },
+];
+
+let isDesktopScreen = window.matchMedia('(min-width: 768px)').matches;
+
+function createArticleRecentWork(
+  name, description, { featuredImageMobile, featuredImageDesktop }, technologies,
+) {
+  const article = document.createElement('article');
+  article.setAttribute('class', 'recent-work');
+
+  const divMostRecent = document.createElement('div');
+  divMostRecent.setAttribute('class', 'most-recent');
+
+  const h2Title = document.createElement('h2');
+  h2Title.setAttribute('class', 'title');
+  const spanTitle = document.createElement('span');
+  spanTitle.textContent = name;
+  h2Title.appendChild(spanTitle);
+
+  const hrCustom = document.createElement('hr');
+  hrCustom.setAttribute('class', 'custom-hr');
+
+  divMostRecent.appendChild(h2Title);
+  divMostRecent.appendChild(hrCustom);
+
+  const divRecentProjects = document.createElement('div');
+  divRecentProjects.setAttribute('class', 'recent-projects');
+  divRecentProjects.setAttribute('id', 'recent-projects');
```

Comment:
- [ ] Your design needs more works on it as it is currently breaking on  desktop screens.take a look at the screenshots :point_down: 
![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/90222110/ddc904b5-2f49-4b17-a31f-42ebcb0a72e2)

Please improve it in all ways so as to match the design given :pray:

Reply:
- [ ] Still, design is not match the original design in, please fix it
![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/38635158/ab3ca52d-96fd-4c52-83f3-e0a7dc445460)
![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/38635158/1ad182eb-3382-41b9-9607-0ba55bcefd76)

- [ ] I am not able to click the other projects 'see project button' kindly fix it